metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
ok, I see.  What I did is to use the existing configurations that were present before, and add configurations for those that did not have any.

So the bro configuration is what metron pre-777 has as the bro configuration.

I will change the default configurations to be whatever makes the most sense.  I did not try to come up with the best default.